tests/test_fields.py

```diff
@@ -63,15 +63,15 @@ def test_validate_length(self):
         self.assertRaises(errors.FieldValueError, field_format.validate_length, "12")
         self.assertRaises(errors.FieldValueError, field_format.validate_length, "123456")
 
-    def test_empty_and_length_limit(self):
+    def test_can_create_empty_field_with_length_limit(self):
         field_format = fields.AbstractFieldFormat("x", True, "3:5", "", _ANY_FORMAT)
         field_format.validate_length("")
 
-    def test_validate(self):
+    def test_can_validate_field_format(self):
```

Comment:
Tests for error conditions should be called `test_fails_on_<error condition>`.

However, there is no point in testing for a `NotImplementedError` as this apparently is a known issue already.

If you really want to leave a not about a future test, just add a `# TODO: test this or that` comment.